docs/InitialAwsSetup.md

```diff
@@ -1,13 +1,122 @@
 # Initial AWS Account Setup
 
+## Requirements
+
+### Workstation
+You will need a Linux machine installed with the following software packages:
+* Python 3.5
+* pip install virtualenv virtualenvwrapper
+* Packer ([download](https://www.packer.io/)) Version 1.1.1 (add it to your path)
+* NodeJS ([download](https://nodejs.org/en/download/)) Version 6.10.X (add it to your path)
+
+## Starting IAM User
+* Manually create IAM user with API keys and full permissions in AWS Console
+
+## Setup name for default VPC
+* in AWS Console under VPCs - Your VPCs - fill in the name "default-vpc" to the default vpc.
+
+## Setup a primary bastion server
+While not necessary for operation it.  It is often nice to have primary 
+bastion server that you can direct to only allow access from your companies 
+IP address range.  Then you can put a hole in your corporate firewall for port 22 
+to the bastion server. This is helpful when creating multiple bossdb stacks, example
+production, develop and testing stacks.  
+
+### Create a bastion key pair.
+* In the AWS Console under EC2 - Key Pairs: "Create Key Pair" for the bastion server
+
+### Create a new key pair for the first bosslet
+* In the AWS Console under EC2 - Key Pairs: "Create Key Pair" for the <your_bosslet_name>
+
+Move both key pairs to the ~/.aws directory. Change the name to end in .pem instead of .pem.txt if that is what it currently is.  
+Change the properties of both pem files to 400
+```sh
+chmod 400 <name>.pem
+``` 
+
+### Create new instance for Bastion
+To create a bastion server create a new t3.micro with thest latest release of ubuntu 14.04, 
```

Comment:
It doesn't have to be an Ubuntu based machine, they just need to set the appropriate username in the bosslet config.